src/pages/VolunteerDashboardPage.jsx

```diff
@@ -71,6 +78,33 @@ const VolunteerDashboardPage = () => {
     }
   }, []);
 
+  useEffect(async () => {
+    try {
+      const [notificationsRes] = await Promise.all([
+        OCHBackend.get('/notification/', { withCredentials: true }),
+      ]);
```

Comment:
Since you're only getting 1 thing, you probably don't need to use `Promise.all`. Just `const notificationRes = await OCHBackend` should be fine